api/envoy/extensions/filters/http/dynamic_forward_proxy/v3/dynamic_forward_proxy.proto

```diff
@@ -30,7 +30,7 @@ message FilterConfig {
 
   // When this flag is set, the filter will add the resolved upstream address in the filter
   // state. The state should be saved with key
-  // `envoy.rbac.matcher.upstream_address_set` (See
+  // `envoy.request.upstream_address_set` (See
```

Comment:
To enter the bikeshed fray I would personally call this `envoy.stream.upstream_address_set` but I would get an ack from @alyssawilk @htuch @ggreenway before changing. :)

Reply:
uh ho :)

Reply:
@mattklein123 @ggreenway @htuch @alyssawilk any preferences between 

- `envoy.stream.upstream_address_set` 
- `envoy.request.upstream_address_set`

Reply:
Let's just go with `envoy.stream.upstream_address_set` since no one else has commented, thanks.